Customer Support Procedures/Neo4j-v5.0-migration.md

```diff
@@ -0,0 +1,47 @@
+<!-- omit in toc -->
+# Neo4j migration to v5 guidelines
+
+If you are planning to migrate your Neo4j database from an older version to v5,
+this document will provide you extra details of what you need to take care of in your Linkurious Enterprise project.
+
+<!-- omit in toc -->
+# Table of contents
+- [Pre-requisites and limitations](#pre-requisites-and-limitations)
+- [Issues with cypher queries](#issues-with-cypher-queries)
+- [Migrating databases](#migrating-databases)
+
+# Pre-requisites and limitations
+
+- Linkurious Enterprise supports Neo4j v5 starting from the version v4.0.1, if you are working with an older version of Linkurious Enterprise
```

Comment:
If you are planning to release it now (before the actual release), I would change 
[...] starting from the version v4.0.1,
to
[...] starting from the version v4.0.1 (estimated release in January 2023),

Reply:
This will actually allow as to start sharing it with customers already asking for the support of Neo4j v5

Reply:
[...] starting from the version v4.0.1 (a beta release estimated to be available in January 2023),

Reply:
Works for me, I've updated this section.